docker-compose.yml

```diff
@@ -79,10 +81,13 @@ services:
     image: gcr.io/trillian-opensource-ci/ctfe
     volumes:
       - ctfeConfig:/etc/config/:ro
+      - ./config/tls:/config/tls:z
     command: [
         "--log_config" ,"/etc/config/ct_server.cfg",
         "--log_rpc_server", "trillian-log-server:8096",
         "--http_endpoint", "0.0.0.0:6962",
+        "--tls_certificate", "/config/tls/tls.crt",
```

Comment:
Could you add a comment with the openssl command you used to generate the crt and key?

Reply:
Since we need several commands to generate key/certs, I've added a small readme file `config/tls/key_cert_generation.md`